liqd_lunch/frontend/src/RestaurantsList.js

```diff
@@ -64,6 +81,14 @@ class  RestaurantsList  extends  Component {
                     <td>
                     <button  onClick={(e)=>  this.handleDelete(e,c.pk) }> Delete</button>
                     <a  href={"/restaurant/" + c.pk}> Update</a>
+                    <button  onClick={(e)=>  this.handleResistanceCreate(e,c.pk) }> Resist</button>
+                    <form onSubmit={this.handleResistanceCreate}>
+                      <div className="form-group">
+                        <input className="form-control" type="text" ref='resistance'/>
+
+                        <input className="btn btn-primary" type="submit" value="Submit" />
+                        </div>
+                      </form>
```

Comment:
Here is the bit that should be one slider (or a textbox and a submit for now). But it needs the submitted number and the pk of the restaurant and I am unsure what to use or how to get that in. @phillimorland